src/serverHistory.ts

```diff
@@ -45,8 +45,12 @@ class ServerHistory implements History {
   complete() {
     this._completeCallback();
   }
+
+  getCurrentLocation() {
+    return this.currentLocation;
+  }
 }
 
-export function createServerHistory(): History {
-  return new ServerHistory();
+export function createServerHistory(currentLocation: Location | Pathname): History {
+  return new ServerHistory(createLocation(currentLocation));
```

Comment:
I think this could actually break applications without this. Currently the application will get passed `undefined` as its first value, where we don't have the same access as below where we can use `drop(1)`/`skip(1)`